cmd/exporter.go

```diff
@@ -151,6 +152,7 @@ func main() {
 	config.SetEnabledHardwareCounterMetrics(*exposeHardwareCounterMetrics)
 	config.SetEnabledGPU(*enableGPU)
 	config.EnabledMSR = *enabledMSR
+	config.SetKubeConfig(*kubeconfig)
```

Comment:
please add in-cluster config support too

Reply:
if kubeconfig is empty we use the in-cluster configuration